### ISSUE TITLE
Remove npm from gradle build

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#18521]
It turns out we do not need to run npm when building dotCMS - all we need to do is use wget to get the package by version, `tar -zxvf` it and then mv the `package` folder to `dotAdmin`
```
wget https://registry.npmjs.org/dotcms-ui/-/dotcms-ui-VERSION.tgz
```
This will help speed up our build process by 30 seconds or so.

Also, bash script to get the latest:
```sh
curl -s https://registry.npmjs.org/dotcms-ui/ |  python -c 'import json,sys;print json.load(sys.stdin)["dist-tags"]["latest"]'
```

```
gradle dotCMSUI{
 doLast{
  def json = new JsonSlurper().parseText("https://registry.npmjs.org/dotcms-ui/")
  def labels = json.properties.dist-tags.latest 
 }
}
```
This needs to happen for `dotcms-ui`
and 
the `dotcms-webcomponents`.

Remove NPM and the npm plugin

**Comment by dsilvam** [comment]
PR: https://github.com/dotCMS/core/pull/18954

**Comment by dsilvam** [comment]
Not working on Docker: 

![](https://files.slack.com/files-pri/T028R2630-F01838N2ZD1/screen_shot_2020-07-31_at_12.46.55_pm.png)

**Comment by dsilvam** [comment]
PR: https://github.com/dotCMS/core/pull/19021

**Comment by dsilvam** [comment]
Passed Internal QA: looking good on Docker.

**Comment by bryanboza** [comment]
Fixed, tested on release-5.3.7 // Postgres // FF and works for me in docker and locally
